packages/core/src/codewhisperer/commands/startTransformByQ.ts

```diff
@@ -191,12 +191,6 @@ export async function preTransformationUploadCode() {
     } catch (err) {
         const errorMessage = `Failed to upload code due to ${(err as Error).message}`
         getLogger().error(errorMessage)
-        telemetry.codeTransform_logGeneralError.emit({
```

Comment:
We were already emitted a metric for this, these were duplicate and unnecessary.